README.md

```diff
@@ -1,68 +1,873 @@
+
+    ```javascript
+    // Specifying a single responseType
+    authParams: {
+      responseType: 'token'
+    }
+
+    // Use an array if specifying multiple response types - in this case,
+    // the response will contain both an ID Token and an Access Token.
+    authParams: {
+      responseType: ['id_token', 'token']
+    }
+    ```
+
+- **authParams.scopes:** Specify what information to make available in the returned `id_token` or `access_token`. For OIDC, you must include `openid` as one of the scopes. Defaults to `['openid', 'email']`.
+
+    Valid OIDC scopes: `openid`, `email`, `profile`, `address`, `phone`
+
+    ```javascript
+    authParams: {
+      scopes: ['openid', 'email', 'profile', 'address', 'phone']
+    }
+    ```
+
+- **authScheme:** Authentication scheme for OIDC authentication. You will normally not need to override this value. Defaults to `OAUTH2`.
+
+    ```javascript
+    authParams: {
+      authScheme: 'OAUTH2'
+    }
+    ```
+
+## Bootstrapping from a recovery token
+
+- **recoveryToken:** Bootstrap the widget into continuing either the Forgot Password or Unlock Account flow after the recovery email has been sent to the user with the `recoveryToken`.
+
+    ```javascript
+    recoveryToken: 'x0whAcR02i0leKtWMZVc'
+    ```
+
+## Feature flags
+
+Enable or disable widget functionality with the following options. Some of these features require additional configuration in your Okta admin settings.
+
+```javascript
+// An example that enables the autoPush and multiOptionalFactorEnroll features
+features: {
+  autoPush: true,
+  multiOptionalFactorEnroll: true
+}
+```
+
+- **features.router** - Set to `true` if you want the widget to update the navigation bar when it transitions between pages. This is useful if you want the user to maintain their current state when refreshing the page, but requires that your server can handle the widget url paths. Defaults to `false`.
+
+- **features.rememberMe** - Display a checkbox to enable "Remember me" functionality at login. Defaults to `true`.
+
+- **features.autoPush** - Display a checkbox to enable "Send push automatically" functionality in the MFA challenge flow. Defaults to `false`.
+
+- **features.smsRecovery** - Allow users with a configured mobile phone number to recover their password using an SMS message. Defaults to `false`.
+
+- **features.callRecovery** - Allow users with a configured mobile phone number to recover their password using a voice call. Defaults to `false`.
+
+- **features.windowsVerify** - Display instructions for enrolling a windows device with Okta Verify. Defaults to `false`.
+
+- **features.selfServiceUnlock** - Display the "Unlock Account" link to allow users to unlock their accounts. Defaults to `false`.
+
+- **features.multiOptionalFactorEnroll** - Allow users to enroll in multiple optional factors before finishing the auth flow. Default behavior is to force enrollment of all required factors and skip optional factors. Defaults to `false`.
```

Comment:
If you're going to use "auth flow" throughout, please use "authentication flow (auth flow)" the first time. Or spell it out everywhere.